lib/stm32wba/hci/README

```diff
@@ -116,7 +118,11 @@ Patch List:
 			  main.h
 			  app_conf.h
 			  scm.c
+			  ll_intf_cmn.h
+			  ll_sys_if.c
+			  linklayer_plat.c
 
 	* Changes from official delivery:
 	- dos2unix applied
 	- trailing white spaces removed
+	- #ifndef __ZEPHYR__ applied to remove Cube specific implementation
```

Comment:
I'd prefer a dedicated bullet for this point which helps identifying the impacted files:
```
	* #ifndef __ZEPHYR__ applied to remove Cube specific implementation
	  Impacted file:  ll_sys_if.c
	                         linklayer_plat.c
```

Reply:
Fixed